<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerSchemeWidget</class>
 <widget class="QWidget" name="PowerSchemeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <widget class="QGroupBox" name="settingGroupBox">
     <property name="title">
      <string>Settings for power scheme</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="computerLabel">
        <property name="text">
         <string>When computer is:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dcPowerLabel">
        <property name="text">
         <string>Plugged in:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="monitorLabel">
        <property name="text">
         <string>Turn off monitor:</string>
        </property>
        <property name="buddy">
         <cstring>monitorDcComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="monitorDcComboBox">
        <item>
         <property name="text">
          <string>After 1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="monitorAcComboBox">
        <item>
         <property name="text">
          <string>After 1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hardDiskLabel">
        <property name="text">
         <string>Turn off hard disks:</string>
        </property>
        <property name="buddy">
         <cstring>hardDiskDcComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="hardDiskDcComboBox">
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="hardDiskAcComboBox">
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="Line" name="schemeSettingsLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="standbyLabel">
        <property name="text">
         <string>System stand by:</string>
        </property>
        <property name="buddy">
         <cstring>standbyDcComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="standbyDcComboBox">
        <item>
         <property name="text">
          <string>After 1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="standbyAcComboBox">
        <item>
         <property name="text">
          <string>After 1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="hibernateLabel">
        <property name="text">
         <string>System hebirnations:</string>
        </property>
        <property name="buddy">
         <cstring>hibernateDcComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QComboBox" name="hibernateDcComboBox">
        <item>
         <property name="text">
          <string>After 1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="hibernateAcComboBox">
        <item>
         <property name="text">
          <string>After 1 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 10 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 15 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 20 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 25 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 30 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 45 mins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 3 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After 5 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="acPowerLabel">
        <property name="text">
         <string>Running on batteries:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="powerGroupBox">
     <property name="title">
      <string>Power schemes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="defaultCheckBox">
        <property name="text">
         <string>Make this the active Power Scheme</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="nameComboBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Home/Office desk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Portable/Laptop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Presentation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always ON</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minimal Power Management</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Max Battery</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>92</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="buddy">
        <cstring>actionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <property name="currentText">
        <string>Create</string>
       </property>
       <item>
        <property name="text">
         <string>Create</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>nameComboBox</tabstop>
  <tabstop>defaultCheckBox</tabstop>
  <tabstop>monitorDcComboBox</tabstop>
  <tabstop>monitorAcComboBox</tabstop>
  <tabstop>hardDiskDcComboBox</tabstop>
  <tabstop>hardDiskAcComboBox</tabstop>
  <tabstop>standbyDcComboBox</tabstop>
  <tabstop>standbyAcComboBox</tabstop>
  <tabstop>hibernateDcComboBox</tabstop>
  <tabstop>hibernateAcComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
